tuned_lens/nn/unembed.py

```diff
@@ -185,7 +105,7 @@ def invert(
 
         if h0 is None:
             # Initialize with the Moore-Penrose pseudoinverse
-            h0 = logits @ self.U_pinv.mT
+            h0 = th.zeros((*leading_dims, d_model), device=logits.device)
```

Comment:
I'm pretty sure it would still be better to initialize with the pseudoinverse even if the norm isn't rolled in... would be easy to test

Reply:
OTOH, idk, do whatever you want.